WooCommerce/Classes/ViewRelated/Editor/AztecAIViewFactory.swift

```diff
@@ -7,10 +7,10 @@ struct AztecAIViewFactory {
     /// - Returns: View that contains the Jetpack AI CTA.
     func aiButtonNextToFormatBar(onTap: @escaping () -> Void) -> UIView {
         let configuration: UIButton.Configuration = {
-            var configuration = UIButton.Configuration.filled()
-            configuration.image = .magicWandIcon(size: Layout.AIButton.imageSize)
+            var configuration = UIButton.Configuration.borderless()
+            configuration.image = .sparklesImage
             configuration.imagePadding = 0
-            configuration.background.backgroundColor = .tertiaryLabel
+            configuration.baseForegroundColor = .brand
```

Comment:
nit: how about using the `accent` color for better contrast in dark mode, like in the `UIButton.applyLinkButtonStyle`? `brand` is purple-60 in both light and dark modes.

<img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/4917d644-d1b1-4f92-8674-670859c76195" width="300" />

Reply:
Makes sense - updated in 944e3ace1f56eedb9f697751cd351995e761db31.